deployment/mainnet/l2addresses.json

```diff
@@ -0,0 +1,17 @@
+{
+  "L2ClaimContract": "0x0000000000000000000000000000000000000000",
+  "L2ClaimImplementation": "0x0000000000000000000000000000000000000000",
```

Comment:
Any reason not to put [0x60790Dc2d45BaA8B36282889569BbB301F4D0c41](https://blockscout.lisk.com/address/0x60790Dc2d45BaA8B36282889569BbB301F4D0c41) here?

Reply:
Same as above.
Done.